docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3"
+services:
+  db:
+    image: postgres
+    container_name: eduplatform_db
+    volumes:
+      - ./data/db:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_DB=POSTGRES_DB
+      - POSTGRES_USER=POSTGRES_USER
+      - POSTGRES_PASSWORD=POSTGRES_PASSWORD
+  web:
+    build: .
+    container_name: eduplatform_web
+    command: >
+      bash -c "python Eduplatform_site/manage.py migrate
+      && python Eduplatform_site/manage.py runserver 0.0.0.0:8000"
```

Comment:
also add `py manage.py makemigrations `  and these commands should be in dockerfile not in compose